model/cluster_request.go

```diff
@@ -165,6 +165,30 @@ func (p *PatchUpgradeClusterRequest) Validate() error {
 		return errors.Errorf("unsupported cluster version %s", *p.Version)
 	}
 
+	if p.RotatorConfig != nil {
+		if p.RotatorConfig.UseRotator == nil {
+			return errors.Errorf("Rotator config use rotator should be set")
```

Comment:
nit: lowecase `rotatator config use....` for this return error and the ones below.